.github/workflows/deploy-github-pages-prod.yml

```diff
@@ -1,8 +1,8 @@
-name: Deploy Github Pages
+name: Deploy Github Pages (Production)
 
 on:
   push:
-    branches: [ main ]
+    branches: [ stable ]
```

Comment:
I actually don't have any preference here, but our github pages deployment is mostly stage/dev. We only added it as a way to show to people who are not able to access the stage deployment. I wouldn't make a separate stable deployment for this config